packages/database/src/migrations/20210720001222-AddKoBoToDataServiceTypes-modifies-schema.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
```

Comment:
Nah, sync groups also need to know what service they're running against, and it made sense to use the same enum (in case in future we have a service that requires both a sync group and one of the other data source types)

Reply:
Right!